pkg/component/gardener/apiserver/apiserver_test.go

```diff
@@ -584,6 +584,7 @@ var _ = Describe("GardenerAPIServer", func() {
 									"--log-format=" + logFormat,
 									"--secure-port=8443",
 									"--workload-identity-token-issuer=" + workloadIdentityIssuer,
+									"--shoot-admin-kubeconfig-max-expiration=4320h",
```

Comment:
move this one line, then the test should be fixed.

Reply:
Please add fix commit as fixup to related commit!

Reply:
done